src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let count = 0;
+  const adder = (numbers) => {
```

Comment:
it's only 1 number, why you're using plural name here?
```suggestion
  let count = 0;
  
  const adder = (number) => {
```